Gemfile

```
@@ -86,7 +95,8 @@ gem 'htmltoword'
 group :development, :test do
   gem 'rspec-rails'
   gem 'rspec'
-  gem 'selenium-webdriver'
+  # pinned this version to work with Ruby 2.1 and Rails 3
+  gem 'selenium-webdriver', '=3.141.0'
```

```suggestion
  gem 'selenium-webdriver', '3.141.0'
```

`=` is the default. https://www.devalot.com/articles/2012/04/gem-versions.html#available-operators What do you think about leaving this out?

---

@pgwillia I plan to close this PR if all the updates are handled individually with dependabot or separate PRs. Many changes included here have already been addressed elsewhere. Also, I agree, we should leave this one out.

---

Gotcha!  The case of too many open pull requests ;)